system_baseline/views/v1.py

```diff
@@ -365,6 +366,20 @@ def _check_for_existing_display_name(display_name, account_number):
         )
 
 
+def _check_for_whitespace_in_display_name(display_name):
+    """
+    check to see if the display name has leading or trailing whitespace
+    """
+
+    if display_name and (
+        display_name != display_name.lstrip() or display_name != display_name.rstrip()
```

Comment:
here is the line you could call the whitespace check again, but just for display_name